File: app/src/main/java/com/mnj/java_solutions/array/ConcatenationOfArrayLC1929.java
package com.mnj.java_solutions.array;

public class ConcatenationOfArrayLC1929 {

    public void concat() {
        int[] nums = {1, 2, 3};

        int res[] = new int[2 * nums.length];

        for (int i = 0; i < nums.length; i++) {
            res[i] = nums[i];
            res[i + nums.length] = nums[i];
        }

        for (int i : res) {
            System.out.println("==>> :" + i);
        }
    }
}

File: app/src/main/java/com/mnj/java_solutions/array/ConcatenationOfArrayLC1929.kt
package com.mnj.java_solutions.array

class ConcatenationOfArrayLC1929 {
    fun concat() {
        val nums = intArrayOf(1, 2, 3)
        val res = IntArray(2 * nums.size)
        for (i in nums.indices) {
            res[i] = nums[i]
            res[i + nums.size] = nums[i]
        }
        for (i in res) {
            println("==>> :$i")
        }
    }
}